src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -4397,21 +4397,22 @@ void ComputeIntSWAbsorpFactors(EnergyPlusData &state)
     } // End of enclosure loop
 }
 
-void ComputeDifSolExcZonesWIZWindows(EnergyPlusData &state, int const NumberOfEnclosures) // Number of solar enclosures
+void ComputeDifSolExcZonesWIZWindows(EnergyPlusData &state)
```

Comment:
Dumped the second parameter, because that's already known in state.

Reply:
Love it :heart: